### PR TITLE
PMM-10092 Grafana 9 Main Menu

### PR DESCRIPTION
**Pull request by matejkubinec**
https://jira.percona.com/browse/PMM-10092

### CODE REVIEW

File: packages/grafana-ui/src/components/Icon/utils.ts [hunk 1]
```diff
@@ -27,6 +27,14 @@ const alwaysMonoIcons: IconName[] = [
...
   'percona-setting',
   'percona-database-checks',
+  'percona-system',
+  'percona-nav-overview',
+  'percona-nav-summary',
+  'percona-database-mysql',
+  'percona-database-postgresql',
+  'percona-database-mongodb',
+  'percona-database-proxysql',
+  'percona-database-haproxy',
```
Comment (fabio-silva) [inline packages/grafana-ui/src/components/Icon/utils.ts:37]: let's add these to `iconBundle.ts`, along with all others 👍

File: public/app/core/components/NavBar/NavBar.tsx [hunk 2]
```diff
@@ -120,32 +128,32 @@ export const NavBar = React.memo(() => {
...
-    if (sttEnabled) {
-      coreItems.push(PMM_STT_PAGE);
-    }
+    if (isAuthorized) {
+      updatedNavTree = buildInventoryAndSettings(updatedNavTree);
 
-    if (dbaasEnabled) {
-      coreItems.push(PMM_DBAAS_PAGE);
-    }
+      if (alertingEnabled) {
```
Comment (fabio-silva) [inline public/app/core/components/NavBar/NavBar.tsx:145]: the indentation seems wrong here

File: public/app/core/components/NavBar/NavBarMenu.tsx [hunk 3]
```diff
@@ -359,33 +385,45 @@ function CollapsibleNavItem({
...
           isOpen={Boolean(sectionExpanded)}
-          onToggle={(isOpen) => setSectionExpanded(isOpen)}
-          className={styles.collapseWrapper}
-          contentClassName={styles.collapseContent}
+          onToggle={handleToggle}
+          className={cx(styles.collapseWrapper, isRoot && styles.rootCollapseWrapper)}
+          contentClassName={cx(styles.collapseContent, isRoot && styles.rootCollapseContent)}
           label={
             <div className={cx(styles.labelWrapper, { [styles.primary]: isActive })}>
+              {!isRoot && <span className={styles.collapsibleIcon}>{getLinkIcon(link, isRoot ? 'xl' : 'sm')}</span>}
```
Comment (fabio-silva) [inline public/app/core/components/NavBar/NavBarMenu.tsx:426]: the span will only show if `!isRoot`, so I believe that `isRoot ? 'xl' : 'sm'` condition is redundant

**Review by fabio-silva — CHANGES_REQUESTED** [review]
Very good job @matejkubinec 🚀 
Just some fome few nits, but it seems working fine !

**Review by fabio-silva — APPROVED** [review]

**Review by solovevayaroslavna — APPROVED** [review]

**Pull request merged**